install/kubernetes/cilium/values.yaml.tmpl

```
@@ -504,6 +504,31 @@ daemon:
   # -- Configure where Cilium runtime state should be stored.
   runPath: "/var/run/cilium"
 
+  # -- Configure a custom list of possible configuration override sources
+  # The default is "config-map:cilium-config,cilium-node-config". For supported
+  # values, see the help text for the build-config subcommand.
+  # Note that this value should be a comma-separated string.
+  configSources: ~
```

nit: Consider adding a [helm-docs](https://github.com/norwoodj/helm-docs#default-valuescolumn) `@default` annotation.

```suggestion
  # @default -- "config-map:cilium-config,cilium-node-config"`
  configSources: ~
```

Also, we could consider making this Helm value of type array instead of type string, and join the values to a string in the template. But I'm fine with it being a string.

---

I'll have a small follow-up helm PR, I'll add the `@default` there. I can't say I have an opinion about array versus string.